legislation_analysis/clustering/service.py

```diff
@@ -85,7 +87,7 @@ def run_knn_clustering() -> None:
         "Starting K-Nearest Neighbor clustering for SCOTUS decisions..."
     )
     scotus_knn = KNN(
-        SCOTUS_DATA_FILE_POS_TAGGED, SCOTUS_DATA_FILE_CLUSTERED_NAME
+        SCOTUS_DATA_CLUSTERED_FILE, SCOTUS_DATA_CLUSTERED_FILE_NAME
```

Comment:
These changes make it so that we are constantly adding to the same file instead of overwriting it every time we cluster.